### PR TITLE
feat: add of deployment(server/client)

### PR DESCRIPTION
**Pull request by devvv1337**

### CODE REVIEW

File: models/FHEModel.py
Comment (CirSandro) [inline models/FHEModel.py:1]: translate FR comments to EN

File: src/client/client.py
Comment (CirSandro) [inline src/client/client.py:1]: translate FR comments to EN

File: src/server/server.py
Comment (CirSandro) [inline src/server/server.py:1]: translate FR comments to EN

File: tests/test_api.py
Comment (CirSandro) [inline tests/test_api.py:1]: translate FR comments to EN

**Review by CirSandro — APPROVED** [review]

**Review by Dixtawi — APPROVED** [review]

**Pull request merged**